file_completer.go

```diff
@@ -0,0 +1,176 @@
+package kong
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// CompleteDirs will search for directories in the given started to be typed path.
+//
+// If no path was started to be typed, it will complete to directories in the current working directory.
+func CompleteDirs() CompleterFunc {
```

Comment:
AHA! I was going to ask why we're implementing all this, but then I remembered you were talking about removing the dependency. LGTM